test/unit/utils-spec.ts

```diff
@@ -154,6 +154,63 @@ describe("Utils", () => {
             }).to.not.throw();
         });
     });
+
+    describe("durationFromString()", () => {

```

Comment:
These unit tests were mostly derived from the examples on the .NET serialization docs here: https://learn.microsoft.com/en-us/dotnet/api/system.timespan.tostring?view=net-7.0#system-timespan-tostring